packages/ui/src/components/templates/ObjectCard/ObjectCard.tsx

```diff
@@ -165,6 +159,12 @@ const ObjectCard = ({
       return null;
     }
 
+    if (item.customComponent != null) {
+      return React.cloneElement(item.customComponent, {
+        key: `${item.displayText} - ${item.order}`,
+      });
+    }
+
```

Comment:
I'm not sure it's a good idea to move it this way. 
I understand the need of having the hideIf being evaluate as a first thing but the hideIfNull can cause issue with the custom component as displayText and indicatorText are not used. Maybe we can split condition into two parts like this : 

```
  if (item.hideIf) {
      return null;
    }

    if (item.customComponent != null) {
      return React.cloneElement(item.customComponent, {
        key: `${item.displayText} - ${item.order}`,
      });
    }

  if (item.hideIfNull &&
        checkNullString(item.displayText) &&
        checkNullString(item.indicatorText)) {
      return null;
    }
```